persistence/src/commonMain/kotlin/com/wire/kalium/persistence/client/SessionStorage.kt

```diff
@@ -47,7 +56,9 @@ interface SessionStorage {
 class SessionStorageImpl(
     private val kaliumPreferences: KaliumPreferences
 ) : SessionStorage {
-    override fun addSession(persistenceSession: PersistenceSession) =
+    private val sessionWasChangedFlow = MutableSharedFlow<Unit>(1).also { it.tryEmit(Unit) }
```

Comment:
I'm thinking if it wouldn't be better to create such Flow solution in `KaliumPreferences` and add for instance `getStringFlow` and `getSerializableFlow` there because if these preferences would be used and updated also outside the `SessionStorage` then we wouldn't get such update here 🤔